src/Security/Authorization/Policy/src/PolicyServiceCollectionExtensions.cs

```diff
@@ -37,7 +37,16 @@ public static IServiceCollection AddAuthorizationPolicyEvaluator(this IServiceCo
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
         public static IServiceCollection AddAuthorization(this IServiceCollection services)
-            => services.AddAuthorization(configure: null);
+        {
```

Comment:
Is there no clean way to share chain these methods anymore?

Reply:
I was following from MVC where it does an explicit null check if you call the overload that takes an option. That unfortunately necessitates having separate code paths.